tests/test_tools.py

```diff
@@ -310,7 +315,7 @@ def test_subexpr_from_unbalanced_parens():
         ]
     for expr, exp in cases:
         obs = subexpr_from_unbalanced(expr, '(', ')')
-        yield assert_equal, exp, obs
+        assert exp == obs
```

Comment:
i can start migrating to pytest, but i think this doesn't prevents us from merging into master, cause each new thing that gets added will have tests in pytest and we wont need to rewrite these aswell.